shopping/views.py

```diff
@@ -1,2 +1,83 @@
+import json
 
-        
+from django.http            import HttpResponse,JsonResponse
+from django.views           import View
+from django.core.exceptions import ObjectDoesNotExist
+from django.db              import transaction
+
+from .models            import *
+from car.models         import ModelVersionLine
+
+class UserView(View):
+    @transaction.atomic
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            DriveContactChannel.objects.create(
+                email  = data['email_privacy'],
+                fax    = data['fax'],
+                mail   = data['mail'],
+                sms    = data['sms'],
+                call   = data['call'],
+                sns    = data['sns']
+            )
+
+            User.objects.create(
+                first_name            = data['first_name'],
+                last_name             = data['last_name'],
+                birthday              = data['birthday'],
+                phone_number          = data['phone_number'],
+                address_city          = AddressCity.objects.get(id = data['city']),
+                address_detail        = data['address'],
+                email                 = data['email'],
+                gender                = Gender.objects.get(id = data['gender']),
+                drive_contact_channel = DriveContactChannel.objects.last(),
+                privacy_check         = data['privacy_check']
+            )
+
+            TestDriveSchedule.objects.create(
+                model_version_line  = ModelVersionLine.objects.get(id = data['mvl']),
+                store_information   = StoreInformation.objects.get(id = data['store']),
+                expect_date         = ExpectDate.objects.get(id = data['expect_date']),
+                contact_us          = data['contact_us']
+            )
+
+            TestDrive.objects.create(
+                test_drive_schedule   = TestDriveSchedule.objects.last(),
+                user                  = User.objects.last(),
+                current_car_model     = data['current_car'],
+            ) 
+
+            return HttpResponse(status = 200)
+
+        except KeyError:
+            print('keyerror')
+            return HttpResponse(status = 400)
+        except ValueError:
+            print('valueerror')
+            return HttpResponse(status = 400)
+        except ObjectDoesNotExist:
+            print('notexist')
+            return HttpResponse(status = 400)
+
+    def get(self, request):
+        cities             = AddressCity.objects.all()
+        exhibition_centers = list(StoreInformation.objects.filter(exhibition_center = 1))
```

Comment:
아래 전시회장은 1번만 보내주는건지 확인을 부탁 드립니다.
```suggestion
        exhibition_centers = list(StoreInformation.objects.filter(exhibition_center = 1))
```

Reply:
해당 컬럼은 booleanField임으로 1로 표기하였는데 True로 변경하겠습니다.